collectors/systemd-journal.plugin/README.md

```diff
@@ -378,31 +395,245 @@ nano /etc/systemd/system/systemd-journal-remote.service
+
+```sh
+sudo chgrp systemd-journal-remote /etc/ssl/private/journal-remote.key
+sudo chgrp systemd-journal-remote /etc/ssl/certs/journal-remote.pem
+sudo chgrp systemd-journal-remote /etc/ssl/ca/trusted.pem
+
+
+sudo chmod 0640 /etc/ssl/private/journal-remote.key
+sudo chmod 755 /etc/ssl/certs/journal-remote.pem
+sudo chmod 755 /etc/ssl/ca/trusted.pem
+```
+
+6. Edit the `systemd-journal-remote.conf` to change the predefined key place and enable SSL.
+
+
+sh```
+sudo nano /etc/systemd/journal-remote.conf
+```
+
+You need to transform the corresponding section to something like this
+```
+[Remote]
+Seal=false
+SplitMode=host
+ServerKeyFile=/etc/ssl/private/journal-remote.key
+ServerCertificateFile=/etc/ssl/certs/journal-remote.pem
+TrustedCertificateFile=/etc/ssl/ca/trusted.pem
+```
 
 
 #### Configuring journal clients to push their logs to the server
 
-On the clients you want to centralize their logs, install `systemd-journal-remote`, configure `systemd-journal-upload`, enable it and start it with `systemctl`.
+In this section we will configure the clients/hosts to push their journal logs into the centralization server. You will install `systemd-journal-remote`, 
+configure `systemd-journal-upload` (with or without SSL), enable and start it.
+
+
+1. Install the `systemd-journal-remote` it run:
 
-To install it run:
 
 ```sh
 # change this according to your distro
 sudo apt-get install systemd-journal-remote
 ```
 
-Then, edit `/etc/systemd/journal-upload.conf` and set the IP address and the port of the server, like this:
+
+2. [With SSL] Copy to the client/hosts the self-signed certificates you created before (you created one per host)
+
+    After this step in a folder (eg `home/user/incoming`), each host must have the following
+    ```
+    clientX.example.com.key
+    clientX.example.com.pem
+    ca.pem #common between the servers
+    ```
+
+
+
+3. [With SSL] _On each client/host;_ create a user and a group (with the same name) called `systemd-journal-upload` 
+
+
+```sh
+sudo adduser --system --home /run/systemd --no-create-home --disabled-login --group systemd-journal-upload
+```
+
+4. [With SSL] _On each client/host;_ Navigate under the folder you placed the certificates, copy them into the expected locations (by the `systemd-journal-upload` service)
+
+
+```sh
+sudo mkdir /etc/ssl/private # make sure that you havent created this folder, and use it already, you may dont want to change it's permissions.
+sudo chmod 755 /etc/ssl/private
+sudo mkdir /etc/ssl/ca/ # make sure that you havent created this folder, and use it already, you may dont want to change it's permissions.
+sudo chmod 755 /etc/ssl/ca
+
+cd home/user/incoming #change it accordingly, this is the place where you copied your certificates.
+sudo cp clientX.example.com.key /etc/ssl/private/journal-upload.key
+sudo cp clientX.example.com.pem /etc/ssl/certs/journal-upload.pem
+sudo cp ca.pem /etc/ssl/ca/trusted.pem
+```
```

Comment:
```suggestion
	```sh
	sudo mkdir /etc/ssl/private # make sure that you havent created this folder, and use it already, you may dont want to change it's permissions.
	sudo chmod 755 /etc/ssl/private
	sudo mkdir /etc/ssl/ca/ # make sure that you havent created this folder, and use it already, you may dont want to change it's permissions.
	sudo chmod 755 /etc/ssl/ca
	
	cd home/user/incoming #change it accordingly, this is the place where you copied your certificates.
	sudo cp clientX.example.com.key /etc/ssl/private/journal-upload.key
	sudo cp clientX.example.com.pem /etc/ssl/certs/journal-upload.pem
	sudo cp ca.pem /etc/ssl/ca/trusted.pem
	```
```